src/helpers/update-tasks.ts

```diff
@@ -1,111 +1,153 @@
+    return false;
+  }
+
+  for (const repo of repos) {
+    logger.info(`Updating reminders for ${repo.owner.login}/${repo.name}`);
+    await updateReminders(context, repo);
   }
+
+  return true;
 }
 
-async function remindAssigneesForIssue(context: Context, issue: Database["public"]["Tables"]["issues"]["Row"]) {
+async function updateReminders(context: Context, repo: ListForOrg["data"][0]) {
   const {
-    adapters: { supabase },
-    logger,
-    config,
+    octokit
   } = context;
-  const now = DateTime.now();
-  const deadline = DateTime.fromISO(issue.deadline);
+  const issues = await octokit.paginate(octokit.rest.issues.listForRepo, {
+    owner: repo.owner.login,
+    repo: repo.name,
+    per_page: 100,
+    state: "open",
+  }) as ListIssueForRepo[];
 
-  if (config.warning <= 0) {
-    logger.info("The reminder threshold is <= 0, won't send any reminder.");
-  } else {
-    const lastReminder = issue.last_reminder;
-    logger.info(`We are passed the deadline on ${issue.url}, should we send a reminder? ${!lastReminder}`);
-    if (!lastReminder && (await remindAssignees(context, issue))) {
-      await supabase.issues.upsert({
-        url: issue.url,
-        deadline: deadline.toJSDate(),
-        lastReminder: now.toJSDate(),
-        lastCheck: now.toJSDate(),
-      });
+  for (const issue of issues) {
+    if (issue.assignees?.length || issue.assignee) {
+      await updateReminderForIssue(context, repo, issue);
     }
   }
 }
 
-async function updateReminders(context: Context, issue: Database["public"]["Tables"]["issues"]["Row"]) {
+async function updateReminderForIssue(context: Context, repo: ListForOrg["data"][0], issue: ListIssueForRepo) {
   const {
-    adapters: { supabase },
     logger,
     config,
     payload,
+    octokit
   } = context;
+  const comments = await octokit.paginate(octokit.rest.issues.listComments, {
+    owner: repo.owner.login,
+    repo: repo.name,
+    issue_number: issue.number,
+    per_page: 100,
+  }) as ListCommentsForIssue[];
+
+  const botComments = comments.filter((o) => o.user?.type === "Bot");
+  const dateRegex = /(?<=<td>)(\w{3}, \w{3} \d{1,2}, \d{1,2}:\d{2} (AM|PM) UTC)(?=<\/td>)/gi;
+  const assignmentRegex = /Ubiquity - Assignment - start -/gi;
+  const botAssignmentComments = sortAndReturn(botComments.filter((o) => assignmentRegex.test(o?.body || "")), "desc");
+  const botFollowup = /this task has been idle for a while. Please provide an update./gi;
+  const botFollowupComments = botComments.filter((o) => botFollowup.test(o?.body || ""));
+
+  if (!botAssignmentComments.length && !botFollowupComments.length) {
+    logger.info(`No assignment or followup comments found for ${issue.url}`);
+    return false;
+  }
+
+  const lastCheckComment = botFollowupComments[0]?.created_at ? botFollowupComments[0] : botAssignmentComments[0];
+
+  const lastCheck = DateTime.fromISO(new Date(lastCheckComment?.created_at).toISOString())
```

Comment:
Is there a reason why you create a JS Date and convert to ISO string? `created_at` is already in ISO format